requirements.txt

```diff
@@ -1,8 +1,8 @@
-Django>=1.8                             # BSD
+Django<2.0                              # BSD
```

Comment:
Why you changing the version of Django?

Reply:
Because I get the following error when I try to install requirements:
```bash
Collecting Django>=1.8 (from -r requirements.txt (line 1))
  Using cached Django-2.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/3k/tyb4z6f50kj_qw7lh354md2m0000gn/T/pip-build-_iBCPd/Django/setup.py", line 32, in <module>
        version = __import__('django').get_version()
      File "django/__init__.py", line 1, in <module>
        from django.utils.version import get_version
      File "django/utils/version.py", line 61, in <module>
        @functools.lru_cache()
    AttributeError: 'module' object has no attribute 'lru_cache'
```

This is because Django 2.0 only supports Python 3.4+, and functools.lru_cache is only available in 3.2+.